packages/xstate-wallet/src/chain.ts

```diff
@@ -199,11 +200,18 @@ export class FakeChain implements Chain {
 
     return merge(first, updates);
   }
-
+  public balanceUpdatedFeed(): Observable<BigNumber> {
```

Comment:
My understanding of having two implementations of the `balanceUpdatedFeed` is that the address parameter is optional. Is it cleaner to have one implementation with an optional parameter?
```suggestion
  public balanceUpdatedFeed(address?: string): Observable<BigNumber> {
```

Reply:
There are actually two classes in here `FakeChain` and `ChainWatcher` which both implement the `Chain` interface. So I needed to add the method to both classes.

Outside the scope of this PR but we should probably move `FakeChain` out of `chain.ts`

Reply:
Ah that makes sense!